corehq/apps/registration/forms.py

```diff
@@ -493,18 +496,28 @@ def __init__(self, data=None, excluded_emails=None, is_add_user=None, location=N
         super(AdminInvitesUserForm, self).__init__(data=data, **kwargs)
         domain_obj = Domain.get_by_name(domain)
         self.fields['role'].choices = role_choices
-        if domain_obj.commtrack_enabled:
+        if domain_obj:
             self.fields['location_id'] = forms.CharField(label='Primary Location', required=False,
-                                                          widget=LocationSelectWidget(domain_obj.name),
-                                                          help_text=EMWF.location_search_help,
-                                                          initial=location.location_id if location else '')
```

Comment:
Why was the initial value removed?

Reply:
nevermind, I see this discussed in the resolve comments

Reply:
Did some more digging and this location seemed to have been added as part of the `ewsghana` project in this [PR](https://github.com/dimagi/commcare-hq/commit/2b74cf7b47fca54ada43ad44f6a259bc7c659ce5#diff-2912c4655bb3c474019c855a831566e13abfe17707fd9fc7b316118de4a233a3). It has since all been deleted but this additions related to the web user invite field and form seemed to have been missed.